### DISCUSSION TITLE
Changing pyart.aux_io.read_d3r_gcpex_nc to pyart.aux_io.read_d3r

### DISCUSSION THREAD

**Discussion by wxwolff** [ARM-DOE/pyart#1349] [General]
NASA' Dual-polarization, Dual-frequency Doppler Rdar (D3R) is currently back online and will soon be deployed to CT to support the IMPACTS field campaign. After February it will return to NASA Wallops Flight Facility as part of the GPM Ground Validation program. I have tested pyart.aux_io.read_d3r_gcpex_nc with the current CF-conformant data and it works well.  I therefore suggest that it makes sense to rename the module with a more general title as pyart.aux_io.read_d3r.

**Comment by mgrover1** [comment]
@wxwolff - would you be willing to share a small subset of one of these files? If have tests for this function, we can move `read_d3r` into the main `pyart.io` module.

**Comment by wxwolff** [comment]
Hi,

Attached is a compressed Ku band D3R file.

Thanks,

Dave



_____^^^^^^^^_______________________________
David B. Wolff, Code 612
GPM Deputy Project Scientist for GV
GPM Ground Validation System Manager
NASA Wallops Flight Facility
34200 Fulton Street
BLDG N159, ROOM E220
Wallops Island, VA 23337
Ph: 757-824-1515, Cell: 443-262-1146
---------------------------------------
////////||||||||\\\\\\\\\\\\\\\\\\



From: Max Grover ***@***.***>
Reply-To: ARM-DOE/pyart ***@***.***>
Date: Friday, December 9, 2022 at 10:40 AM
To: ARM-DOE/pyart ***@***.***>
Cc: "Wolff, David B. (WFF-6120)" ***@***.***>, Mention ***@***.***>
Subject: [EXTERNAL] Re: [ARM-DOE/pyart] Changing pyart.aux_io.read_d3r_gcpex_nc to pyart.aux_io.read_d3r (Discussion #1349)


@wxwolff<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fwxwolff&data=05%7C01%7CDavid.B.Wolff%40nasa.gov%7C4571e408e3e64c0e2c2f08dad9fbbd99%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C638061972494423241%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=nmBhzEVSlzoP1Id7xzFgNf2rBtpnGFxS9zq9LzSBrE8%3D&reserved=0> - would you be willing to share a small subset of one of these files? If have tests for this function, we can move read_d3r into the main pyart.io module.

**Reply by mgrover1** [reply]
Great! Does this sound a reasonable solution?

**Comment by wxwolff** [comment]
Yes, thanks.





_____^^^^^^^^_______________________________
David B. Wolff, Code 612
GPM Deputy Project Scientist for GV
GPM Ground Validation System Manager
NASA Wallops Flight Facility
34200 Fulton Street
BLDG N159, ROOM E220
Wallops Island, VA 23337
Ph: 757-824-1515, Cell: 443-262-1146
---------------------------------------
////////||||||||\\\\\\\\\\\\\\\\\\



From: Max Grover ***@***.***>
Reply-To: ARM-DOE/pyart ***@***.***>
Date: Friday, December 9, 2022 at 11:01 AM
To: ARM-DOE/pyart ***@***.***>
Cc: "Wolff, David B. (WFF-6120)" ***@***.***>, Mention ***@***.***>
Subject: [EXTERNAL] Re: [ARM-DOE/pyart] Changing pyart.aux_io.read_d3r_gcpex_nc to pyart.aux_io.read_d3r (Discussion #1349)


Great! Does this sound a reasonable solution?

**Reply by mgrover1** [reply]
I do not see the file - can you email to mgrover@anl.gov?
